users/views.py

```diff
@@ -74,22 +71,19 @@ def user_profile(request, username):
                 profile_form.save()
                 update_session_auth_hash(request, profile_user)
                 request.user = profile_user
-                render_data['profile_message'] = 'Update successfully'
+                messages.success(request, 'Update profile successfully!')
 
         if request.method == 'POST' and 'userlevel_form' in request.POST:
-            userlevel_form = UserLevelForm(request.POST)
+            userlevel_form = UserLevelForm(request.POST, request_user=request.user)
             if can_change_userlevel(request.user, profile_user):
                 if userlevel_form.is_valid(request.user):
                     user_level = userlevel_form.cleaned_data['user_level']
-                    logger.info('User %s update %s\'s user_level to %s' %
+                    logger.info('User %s update %s\'s user level to %s' %
                                 (request.user, username, user_level))
                     profile_user.user_level = user_level
                     profile_user.save()
-                    render_data['userlevel_message'] = 'Update successfully'
-                else:
```

Comment:
Is there any possibility of sending hide choices to server?
If so, this checking should not be removed.